.github/workflows/test-integration-template.yaml

```diff
@@ -128,124 +114,40 @@ jobs:
-      id: deployment
-      with:
-        step: start
-        token: ${{ steps.generate-github-token.outputs.token }}
-        env: ${{ steps.vars.outputs.identifier }}
-        ref: ${{ inputs.caller-git-ref }}
-    - name: Pre setup
-      timeout-minutes: 5
-      env:
-        TEST_CHART_FLOW: ${{ matrix.scenario.flow }}
-        TEST_INGRESS_HOST: ${{ steps.vars.outputs.ingress-host }}
-      run: |
-        task -d $TEST_SCENARIOS_DIR/chart-full-setup setup.pre
 
-        echo "Extra values from workflow:"
-        echo "${{ inputs.extra-values }}" > /tmp/extra-values-file.yaml
-        cat /tmp/extra-values-file.yaml
-    - name: 🌟 Setup Camunda chart 🌟
-      env:
-        TEST_CHART_FLOW: ${{ matrix.scenario.flow }}
-        TEST_HELM_EXTRA_ARGS: >-
-          --set global.ingress.host=${{ steps.vars.outputs.ingress-host }}
-          --values /tmp/extra-values-file.yaml
-        TEST_PROMETHEUS_HELM_EXTRA_ARGS: "--set host=${{ steps.ingress.outputs.host }}"
-      run: |
-        task -d $TEST_SCENARIOS_DIR/chart-full-setup setup.exec
-    - name: Post setup
-      timeout-minutes: 5
-      run: |
-        task -d $TEST_SCENARIOS_DIR/chart-full-setup setup.post
-    - name: Pre Upgrade
-      if: matrix.scenario.flow == 'upgrade'
-      run: |
-        task -d $TEST_SCENARIOS_DIR/chart-full-setup upgrade.pre
-    - name: 🌟 Upgrade Camunda chart 🌟
-      if: matrix.scenario.flow == 'upgrade'
-      env:
-        TEST_HELM_EXTRA_ARGS: >-
-          --set global.ingress.host=${{ steps.vars.outputs.ingress-host }}
-          --values /tmp/extra-values-file.yaml
-      run: |
-        task -d $TEST_SCENARIOS_DIR/chart-full-setup upgrade.exec
-    - name: Update GitHub deployment status
-      uses: bobheadxi/deployments@648679e8e4915b27893bd7dbc35cb504dc915bc8 # v1
-      with:
-        step: finish
-        token: ${{ steps.generate-github-token.outputs.token }}
-        status: ${{ job.status }}
-        deployment_id: ${{ steps.deployment.outputs.deployment_id }}
-        env_url: https://${{ steps.vars.outputs.ingress-host }}
-        env: ${{ steps.vars.outputs.identifier }}
-        ref: ${{ inputs.caller-git-ref }}
-    - name: ⭐️ Run Preflight TestSuite ⭐️
-      if: inputs.test-enabled
-      timeout-minutes: 10
-      run: |
-        task -d $TEST_SCENARIOS_DIR/chart-full-setup test.preflight
-    - name: ⭐️ Run Core TestSuite ⭐️
-      if: inputs.test-enabled
-      timeout-minutes: 20
+    - name: Ensure identifier is set
       run: |
-        task -d $TEST_SCENARIOS_DIR/chart-full-setup test.core
-    - name: 🚨 Get failed Pods info 🚨
-      if: failure()
-      uses: ./.github/actions/failed-pods-info
-    - name: Cleanup GitHub deployment
-      if: always() && (env.TEST_PERSISTENT == 'false' || matrix.distro.type != 'kubernetes')
-      uses: bobheadxi/deployments@648679e8e4915b27893bd7dbc35cb504dc915bc8 # v1
+        if [ -z "${{ env.TEST_IDENTIFIER }}" ]; then
```

Comment:
Why check the identifier var if the input is mandatory already?

Reply:
cool, then let's add that as a comment above that line so it's easy to understand why it's done that way.